bootcampspringboot/src/main/java/com/max/bootcampspringboot/data/entity/Knowledge.java

```diff
@@ -0,0 +1,75 @@
+package com.max.bootcampspringboot.data.entity;
+
+import jakarta.persistence.*;
+
+@Entity
+@Table(name = "knowledges")
+public class Knowledge {
+    @EmbeddedId
+    private KnowledgeId id;
+
+    @Column(name = "experiencelevel")
```

Comment:
In der Datenbank bitte snake_case

Reply:
ich lass jetzt einfach die datenbank so - bin bissl zu faul das überall dann anzupassen. Theoretisch kann die Datenbank ja auch schon exisiteren und dann muss man halt das extra umbennen wenn jmd das besonder benannt hat in der DB.